tools/nni-auto-tune/search_space.json

```diff
@@ -0,0 +1,12 @@
+{
+    "BKTKmeansK": {"_type": "choice", "_value": [4,8,16,32]},
+    "Samples": {"_type": "choice", "_value": [1000, 2000, 4000, 8000, 10000]},
+    "TPTNumber": {"_type": "choice", "_value": [32, 34, 38, 46, 62, 94, 128]},
+    "RefineIterations": {"_type": "choice", "_value": [2, 3]},
+    "NeighborhoodSize": {"_type": "choice", "_value": [16, 18, 22, 30, 46, 64]},
+    "CEF": {"_type": "choice", "_value": [1000, 1100, 1200, 1400, 1800,2000]},
+    "MaxCheckForRefineGraph": {"_type": "choice", "_value": [4096, 5120, 7168, 11264,16324]},
+    "NumberOfInitialDynamicPivots": {"_type": "choice", "_value": [1, 2, 4, 8, 16,32,50]},
+    "NumberOfOtherDynamicPivots": {"_type": "choice", "_value": [1, 2, 4, 8, 10]}
+}
+
```

Comment:
NNI tuners such as TPE, SMAC, hyperband treat each value of type "choice" not related,  the order of the integers is not leveraged by NNI.  In order to make NNI search parameters more efficiently,  suggest to change parameter type from `choice` to `quniform` or `qloguniform`.
For example,  maybe we can change "CEF"  to :
```
"CEF": {"_type": "quniform", "_value": [1000, 2000, 100]}
```
NNI search spaces: https://nni.readthedocs.io/en/stable/hpo/search_space.html#randint